lib/minimax_rb.rb

```diff
@@ -1,5 +1,9 @@
 module MinimaxRB
+  ARGUMENT_MISSING_MESSAGE = "One or more missing arguments!"
+  EXPECTED_ARGUMENTS = [:board, :curr_player_marker, :opponent_marker]
+
   def self.best_move args
+    raise ArgumentError, ARGUMENT_MISSING_MESSAGE unless all_args_included?(args)
     minimax(args[:board], args[:curr_player_marker], args[:opponent_marker], args.fetch(:empty_marker, nil))
```

Comment:
I may have missed this when I was looking at the code yesterday, but is there a case to handle the opponent winning?  I saw logic for a score of 0 for a draw game and a score for one of the players winning, but shouldn't there be a score for if the opponent wins instead?